### PR TITLE
Project scaffolding and Team intro flow

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes include:

1. Project scaffolding.
2. Handling the Dost team intro flow.

### CODE REVIEW

**Review by rudresh — CHANGES_REQUESTED** [review]
Looks good for scaffolding, one minor feedback.

File: api/helpers/db_helper.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from api import models
+
+# TODO: Need to refactor this and try using ORM
```
[the remark this answers is not in the document]
Comment (Satendra-SR) [reply api/helpers/db_helper.py:3]: Yea! right. I've created a separate [tracking](https://github.com/DostEducation/RP_IVR_analytics/issues/35) for that.

**Review by rudresh — APPROVED** [review]
LGTM 👍

**Comment by Satendra-SR** [comment]
Merging now.

**Pull request merged**